Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1

```diff
@@ -86,6 +86,12 @@ function Main {
 		}
 		$FirstRun = $true
 		Provision-VMsForLisa -AllVMData $AllVMData -installPackagesOnRoleNames "none"
+		foreach ($VmData in $AllVMData) {
+			Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username "root" -password $password "echo $($VmData.RoleName) > /etc/hostname"
```

Comment:
maybe we should remove hardcoded 'root' in the code.

Reply:
will use variable instead